<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunScriptDialog</class>
 <widget class="QDialog" name="RunScriptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Run Script</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="ScFileWidget" name="fileWidget" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="extChk">
     <property name="text">
      <string>Run as Extension Script</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScFileWidget</class>
   <extends>QWidget</extends>
   <header>ui/scfilewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
